src/components/BarChart/Axes/Axes.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const Axes = memo(({ intervals }) => (
+	<div className="chart__axes">
+		{intervals.map((element, index) => (
+			<div className="chart__interval" key={index}>
+				{element}
+			</div>
+		))}
+	</div>
+));
+
+Axes.propTypes = {
+	intervals: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
A single axis "number with line" is called "tick".

The property is better to name "domain"
"Domain" - is a range of values for the axis. [min, max] - for numeric axis. [1st, 2ns, 3rd...] - for named axis.
For your first task, let's use just named axis.